spec/models/project_spec.rb

```diff
@@ -22,6 +22,11 @@
   	expect(Rugged::Repository.new(File.join @project.path , 'satellite' , '.git')).to be_a(Rugged::Repository)
   end
   
+  it "lists public projects for a certain user" do
```

Comment:
How exactly does this test case test "lists public projects for a certain user"? 

It tests the query `Project.where.not(private: true)`. If we wanted to test that, (have this as a unit test for the integration test above), then this query should be moved in to the Project model, so that we're testing the same code that the controller uses :)